spec/UserDbSpec.js

```diff
@@ -13,45 +13,97 @@ describe("User DB", () => {
     */
 
     /* A specification */
-    it("shall be able to retrieve a user with a correct username and password", async function() {
+    it("shall be able to retrieve a user with a correct username and password", async function () {
         let res = await UserDb.addUser("vcedgar", "1234");
-        expect(res).toBe(true);
+        expect(res)
+            .toBe(true);
 
         let result1 = await UserDb.userLogin("vcedgar", "1234");
         let result2 = await UserDb.userLogin("vcedgar", "12345");
         let result3 = await UserDb.userLogin("vcedgar6", "1234");
-        expect(result1).toBe(true);
-        expect(result2).toBe(false);
-        expect(result3).toBe(false);
+        expect(result1)
+            .toBe(true);
+        expect(result2)
+            .toBe(false);
+        expect(result3)
+            .toBe(false);
     });
 
     /* A specification */
-    it("shall be able to add a user with a new username", async function() {
+    it("shall be able to add a user with a new username", async function () {
         let result1 = await UserDb.addUser("zlmonroe", "1234");
         let result1p2 = await UserDb.userLogin("zlmonroe", "1234");
 
-        expect(result1 && result1p2).toBe(true);
+        expect(result1 && result1p2)
```

Comment:
you should not use && for tests usually I believe jasmine should support having two expect statements, why not use that?

Reply:
This is not a hard fix to separate these. I know you didn't write the code originally, but if you could do this while you fix other things that would be nice.